doc/adr/0008-concept-for-handling-ldml.de-in-backend.md

```diff
@@ -0,0 +1,53 @@
+# 6. Concept for handling LDML.de in the backend
+
+Date: 2024-04-12
+
+## Status
+
+Proposed
+
+## Context
+
+We have identified two main issues with our current approach in the backend source code:
+- The current domain model is quite artificial, depicting just one specific use case (a.k.a amending laws).
+- The database model contains data that is duplicated, meaning present within the XML (xml columns) and in a separate column.
```

Comment:
I guess this will not go away as we will still have "eli" and "xml" in our law column even though technically it's data duplication.

Reply:
It is data duplication yes and is the only one we will have, for now. It is documented in the consequences part